packages/cli/lib/imageslice.js

```diff
@@ -0,0 +1,255 @@
+ * It needs to have the images that need to be sliced, living in `static/img/iiif`, so
+ * that the script will look for any images in there to slice and create the folders/zoom depth.
+ * If it happens to detect folders are already there, it'll error out and tell you to remove them
+ */
+export default async function () {
+  return new Promise((resolve) => {
+    let spinner = ora({
+      text: "Creating IIIF images..."
+    }).start();
+    const iiifSeed = "static/img/iiif/images";
+    const iiifProcessed = "static/img/iiif/processed";
+    let images = [];
+    let finished = false;
+    let imageProcessed = false;
+    let iterations = 0;
+    let requests = 0;
+    let iiifTiler = isWin()
+      ? "/go-iiif/bin/iiif-tile-seed-win"
+      : process.platform === "linux"
+      ? "/go-iiif/bin/iiif-tile-seed-linux"
+      : "/go-iiif/bin/iiif-tile-seed-macos";
+    const iiifConfig = "/go-iiif/config/";
+
+    // check if processed folder exists
+    if (!fs.existsSync(iiifProcessed)) {
+      fs.mkdirSync(iiifProcessed, { recursive: true });
+    }
+
+    // Read the config template and create a local state of the config to overwrite
+    // values that will allow it to work per project.  Create file after values set.
+    function updateConfigPath() {
+      spinner.info("Updating go-iiif config");
+      let config = fs.readFileSync(
+        __dirname + path.normalize("/go-iiif/config/config.template")
+      );
+      config = JSON.parse(config);
+      config.images.source.path = path.normalize(
+        process.cwd() + "/" + iiifSeed
+      );
+      config.derivatives.cache.path = path.normalize(
+        process.cwd() + "/" + iiifProcessed
+      );
+      config = JSON.stringify(config, null, 2);
+      fs.writeFile(
+        __dirname + path.normalize("/go-iiif/config/config.json"),
+        config,
+        "utf8"
+      );
+      spinner.succeed("go-iiif config created!");
+    }
+
+    // Log to spinner the process and then remove config file so only the template remains,
+    // Trigger done function
+    function resetConfigPath() {
+      spinner.info("Cleaning up go-iiif config");
+      fs.unlink(__dirname + path.normalize("/go-iiif/config/config.json"));
+      spinner.succeed("go-iiif config cleaned up");
+      done();
+    }
+
+    // function to execute the command to slice the images
+    async function iiifSlice(imageUrl) {
+      const imageName = imageUrl.substring(imageUrl.lastIndexOf("/") + 1);
+      const cmd = `${path.normalize(__dirname)}${path.normalize(
+        iiifTiler
+      )} -config-source file://${path.normalize(__dirname)}${path.normalize(
+        iiifConfig
+      )} -quality default -verbose -refresh -scale-factors 128,64,32,16,8,4,2,1,0 ${imageName}`;
+
+      return new Promise(function (resolve, reject) {
+        // execSync(cmd); // alternative command, more console logs
+        return execa.commandSync(cmd) ? resolve(true) : reject(true);
+      });
+    }
```

Comment:
The below may be easier to read, configure, and maintain however if we have a directory of images this method will be called for each one and should be refactored so that the command is constructed only once and then called with the variable parameter that is the `imageName`

```suggestion
    async function iiifSlice(imageUrl) {
      const imageName = imageUrl.substring(imageUrl.lastIndexOf("/") + 1);
      const configPath = `file://${path.normalize(__dirname)}${path.normalize(iiifConfig)}`
      const cmdArgs = [
        '-quality default',
        '-verbose',
        '-refresh' 
        '-scale-factors 128,64,32,16,8,4,2,1,0' 
        `-config-source ${configPath}`
      ];
      const cmdPath = ${path.normalize(__dirname)}${path.normalize(iiifTiler)};
      const cmdString = `${cmdPath} ${cmdArgs.join(' ')} ${imageName}`;

      return new Promise((resolve, reject) => {
        // execSync(cmdString); // alternative command, more console logs
        return execa.commandSync(cmdString) ? resolve(true) : reject(true);
      });
    }
```